app/src/main/res/layout/activity_rewards_level.xml

```diff
@@ -36,14 +36,8 @@
         layout="@layout/no_network_retry_only_layout"
         android:layout_width="match_parent"
```

Comment:
why do we have a constraint layout inside a constraint layout with only a single child?